### ISSUE TITLE
Merged cuts -> playback pauses/freeze and then resume: pulse audio output debug: underflow & playback way too early

### ISSUE THREAD

**Issue by slrslr** [mifi/lossless-cut#1245]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Try with a different kind of file to confirm that the problem is related to just this file.
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read all the documentation at https://github.com/mifi/lossless-cut and **Known issues / Troubleshooting**
- [X] Search for your problem under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Steps to reproduce

Hello,
3.46.2 AppImage on Debian stable.

I have drag & drop mp4 file ([file detail by ffplay](https://bin.disroot.org/?5caa0f8300efa3f4#4hcWVj3WuKgDSfXr8rzF6LasCFPWuWXxPwXTYG9zZ5jA)), then .srt file, then click Export in aim to merge the subtitles with the video.
MOV faststart ENABLED
Leave MP4 metadata ENABLED
Now i drag&drop resulting video to LC and set two fragments (using key button to jump to key frames) in the first 2 minutes of the video. Click export,
MOV faststart ENABLED (fails also when is DISABLED)
Leave MP4 metadata ENABLED (fails also when is DISABLED)
Keyframe cut
Merge fragments
Play the merged video ([here uploaded](https://drive.proton.me/urls/ARZQD2XVA8#B77D6jQefcop)) using VLC 3.0.17.4 (seems to be latest stable, is from flatpak https://www.videolan.org/vlc/#download ) and it is stuck for many seconds between the fragments, but then resume anyway.

when video freezes, this appear:
```
[00007fc52c04ee30] main filter warning: dropping pictures
[00007fc52c04ee30] main filter warning: dropping pictures
pulse audio output debug: underflow
```

then when it "resume" playback, it shows:
```
[0000556e012e2ed0] main audio output warning: playback way too early (-36128098): playing silence
[00007fc53404bf20] main video output warning: picture is too late to be displayed (missing 34889 ms)
[0000556e012e2ed0] main audio output debug: inserting 1734148 zeroes
[00007fc53404bf20] main video output warning: picture is too late to be displayed (missing 34764 ms)
[00007fc52c04ee30] main filter warning: dropping pictures
```

MPV player does not play the file correctly too (no freeze, but second fragment not played entirely)
ffplay plays it correctly

Is this something to be fixed please?

### Expected behavior

no freeze/pause playback

### Actual behavior

freezes/pauses playback for many seconds then resumes

### Provide an error report

https://bin.disroot.org/?59ddb152669bdff4#Bw4tzpCHHVU4ZzYHF65RqBkZok4agNU2KZ6opbPsr8uf

### Share the file

https://drive.proton.me/urls/ARZQD2XVA8#B77D6jQefcop

### Share log

-

**Comment by mifi** [comment]
Hi!
Sorry for the delay here. I tried to download your source file, and cut/merge it according to your error report (two segments 00:00:09.259 - 00:00:17.434 and 00:00:26.068 - 00:00:29.404)

And the output file does play back successfully for me. Have you tried to experiment with the different settings? Here are mine:

<img width="706" alt="Screenshot 2022-10-14 at 21 38 37" src="https://user-images.githubusercontent.com/402547/195928362-51129a8d-f20e-4d16-9dc9-33cea38137b0.png">

**Comment by slrslr** [comment]
When i set cut points same as yours, then it also plays good for me.

Here is my second attempt so you can quickly reproduce the issue.
Please extract [this 7z archive](https://drive.proton.me/urls/7Z3R1VS620#s8otKcpZnYZq) containing source file and .llc file and details how to export and play in VLC player (i have up to date package from Debian 11 - stable repository).
Thank you

**Comment by cunlem** [comment]
I've had identical problem with the subtitle track type embedded in MP4 file. It's the format used when downloading youtube video with `yt-dlp` and using `--embed-subs` option.

**Comment by mifi** [comment]
@cunlem does it work if you disable the subtitle track like I described?

**Comment by cunlem** [comment]
Yes excluding the subtitle track from export produces file without errors

**Comment by mifi** [comment]
Ok, then I think this is a bug in ffmpeg, and I don't know what I can do in losslesscut other than maybe warning the user that cutting/merging files with subtitles might not work and suggest them to disable subtitles.
